OracleWebCenterContent/dockerfiles/README.md

```diff
@@ -0,0 +1,274 @@
+To add/modify user to be part of docker group
+```
+   $ sudo /sbin/usermod -a -G docker <userid>
+```
+
+## 3.2. Set the Proxy if required:
+
+Set up the proxy for docker to connect to the outside world - to access external registries and build a Docker image, set up environment variables for proxy server something like this:
+```
+   export http_proxy=http://proxy.example.com:80 
+   export https_proxy=http://proxy.example.com:80 
+   export HTTP_PROXY=http://proxy.example.com:80 
+   export HTTPS_PROXY=http://proxy.example.com:80 
+   export NO_PROXY=localhost,.example.com 
+```
+
+## 3.3. Create a User Defined network
+
+In this configuration creation of a user defined network will enable communication between the containers just using container names. User defined network option was preferred over the container linking option as the latter is now deprecated. For this setup we will use a user defined network using bridge driver.Create a user defined network using the bridge driver by executing the following command:
+
+        $ docker network create -d bridge <some name>
+
+Sample command ...
+
+        $ docker network create -d WCContentNetwork
+
+## 3.4. Mount a host directory as a data volume
+You need to mount volumes, which are directories stored outside a container's file system, to store WebLogic domain files and any other configuration files. 
+
+To mount a host directory (`$DATA_VOLUME`) as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
+
+> This ensures 'oracle' user has access to shared volume.
+
+```
+sudo mkdir -p /<YOUR_HOST_DIRECTTORY_PATH>/wccontent
+sudo chown 1000:1000 /<YOUR_HOST_DIRECTTORY_PATH>/wccontent
+```
+
+All container operations are performed as **'oracle'** user.
+
+**Note**: If a user already exists with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+## 3.5. Database
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating WebCenter Content specific RCU schemas while configuring WebCenter Content domain. 
+
+The Oracle Database image can be pulled from [Oracle Container Registry](https://container-registry.oracle.com) or build your own using the [Oracle Database Dockerfiles and scripts]((https://github.com/oracle/docker-images/tree/master/OracleDatabase)) in this repo.
+
+## 3.6. Docker Security Configuration
+
+For detailed instructions of security best practices, please refer to this [documenation](https://docs.oracle.com/en/operating-systems/oracle-linux/docker/docker-security.html#docker-security-components).
+
+# 4. Building Oracle WebCenter Content Image
+
+To build/use a WebCenter Content image you should have Oracle FMW Infrastrucure image.
```

Comment:
You only need the image to build, not to use. And it's the Oracle Fusion Middleware Infrastructure images, not "FMW". You've also misspelled "Infrastructure". Finally, I asked you to remove lines 148-165 and replace them with a link to the Oracle Fusion Middleware Infrastructure README instead.